### ISSUE TITLE
Running on Windows

### ISSUE THREAD

**Issue by bencrossman** [falkTX/Carla#743]
I'm considering switching to Carla as my VST host on PC but having trouble getting my VST's to load.
I'm using Windows 10. 

If I run "Carla_2.0-beta7-win32" I get the python error:

![image](https://user-images.githubusercontent.com/13842596/45584032-e1476280-b90f-11e8-98c0-a9ee01ee0972.png)

I've installed all VC redistributables

If I run Carla_2.0-beta7-win64 it does run and I can get my 32 bit VST plugins to list (in experiential bridge mode) but I can't seem to add any. It just does nothing when I add a listed plugin.

**Comment by falkTX** [comment]
Interesting. first time I see this issue.
I have to make new binaries, the old ones were a bit broken.
Try https://github.com/falkTX/Carla/releases/download/v1.9.9/Carla_2.0-beta7.3-win32-nosse.zip and see if that works for you.

**Comment by bencrossman** [comment]
All my VST's appear to be working except DirectWave. Doesn't get picked up by the scan.
https://www.image-line.com/plugins/Synths/DirectWave/

Anyway not your problem but if you'd like the dll sent let me know.

Cheers
Ben

**Comment by falkTX** [comment]
I know the plugin, I can load it just fine here.
Maybe you have the 64bit version of it, the Carla binary I linked to above was for 32bit only.
In any case, the issue you had is fixed with the latest builds.
I am wrapping up details regarding 2.0-RC1 as release, will do builds quite soon, tag and add a changelog too.

Please test afterwards, on the RC1 builds.
I am closing off this ticket since the original issue reported here is fixed.

**Comment by bencrossman** [comment]
It's definitely x86. I use to use Brainspawn Forte as my VST host solution
but they recently closed down. As a replacement, I am thinking of writing a
setlist / song manager and then pipe the midi from that into a Carla rack.

Will test out RC1

Thanks.

**Comment by bencrossman** [comment]
So win32 RC1 (both sse and non) worked and DirectWave scanned this time. However when I added it the application closes out.

However I was able to run the 64bit version. My 32 bit bridged plugins showed up (not sure if this is because of successfully scanning them using the 32bit version) and DirectWave works great in this scenario along with all my other plugins so this will be the path I go.

Thanks
